core/textinput/src/textinput/Editor.cpp

```diff
@@ -262,10 +283,23 @@ namespace textinput {
         } else {
           ++fCurHistEntry;
         }
+        SetReverseHistSearchPrompt(R.fDisplay);
```

Comment:
I think we don't need this, as the `[bkw'...']` prompt doesn't need updating if the user just pressed Ctrl+R.

Reply:
But wouldn't it need updating from "fwd" to "bkw"  if I switch from CTRL+S to CTRL+R ?

Reply:
Yes, I tried removing that part. If press two times CTRL+R and twice CTRL+S, I get:
```
[bkw'int'] int c
[bkw'int'] int b
[bkw'int'] int a
[bkw'int'] int b
[bkw'int'] int c
```

instead of `fwd` in the last two lines